docs/mine/lotus/split-markets-miners.md

```diff
@@ -8,13 +8,39 @@ breadcrumb: 'Splitting main miner and markets service processes'
 
 {{ $frontmatter.description }}
 
-## Background
+[[TOC]]
+
+## Concepts
+
+Lotus performs mining operations, such as sealing files into sectors, calculating proofs over
+those files and submitting the proofs on chain.
+Lotus also performs markets operations, providing storage and serving retrievals to clients.
+
+It is now possible to run mining and markets operations in separate processes. Service providers can
+accept storage and retrieval deals without impacting ongoing mining operations.
+
+It is recommended to run the mining and markets processes on separate machines so that
```

Comment:
```suggestion
It is recommended to run the mining and markets processes on separate physical or virtual machines so that
```

Reply:
We need to stress the difference between *process* and *machines*. When I first read it, I was a bit confused.